python/mxnet/contrib/serde/_import/backend_rep.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
2018, repeats in more files, please update.

Reply:
I will update all the license and copyright statements.